javascript/javascript3/week2/product.js

```diff
@@ -0,0 +1,89 @@
+class Product {
+  constructor(name, price) {
+    this.name = name;
+    this.price = price;
+  }
+  convertToCurrency(currency) {
+
+    if (currency.toLowerCase().includes('dollar')) {
+      return this.price * 0, 16
+    } else if (currency.toLowerCase().includes('euro')) {
+      return this.price * 0, 13
+    } else if (currency.toLowerCase().includes('pound')) {
+      return this.price * 0, 12
+    } else {
+      return this.price
+    }
+  }
+}
+
+class ShoppingCart {
+  constructor() {
+    this.products = [];
+  }
+
+  addProduct(product) {
+    this.products.push(product)
+  }
+
+  removeProduct(product) {
+    const updatedProducts = this.products.filter((e) => product.name !== e.name)
+    this.products = updatedProducts
+  }
+
+  searchProduct(productName) {
+    return this.products.filter(a => a.name === productName)
+  }
+
+  getTotal() {
+    return this.products.reduce((a, b) => a.price + b.price)
+
+  }
+
+  renderProducts() {
+    const ul = document.querySelector("body");
+    this.products.forEach(element => {
+      const liname = document.createElement("h2");
+      liname.innerHTML = element.name;
+      ul.appendChild(liname)
+
+      const liprice = document.createElement("p");
+      liprice.innerHTML = element.price;
+      ul.appendChild(liprice)
+    });
+  }
+
+  getUser() {
```

Comment:
getUser should return a promise with the data from this api: 
https://jsonplaceholder.typicode.com/users/1